python/gtsam/tests/test_pickle.py

```diff
@@ -0,0 +1,48 @@
+"""
+GTSAM Copyright 2010-2020, Georgia Tech Research Corporation,
+Atlanta, Georgia 30332-0415
+All Rights Reserved
+
+See LICENSE for the license information
+
+Unit tests to check pickling.
+
+Author: Ayush Baid
+"""
+from gtsam import Cal3Bundler, PinholeCameraCal3Bundler, Point2, Point3, Pose3, Rot3, SfmTrack, Unit3 
+
+from gtsam.utils.test_case import GtsamTestCase
+
+class TestPickle(GtsamTestCase):
```

Comment:
I created this single file for all the pickle tests, but I prefer having them in their individual test files.

The problem is that we currently have the python test file for `Pose3` and not for other types. So I might need to create a file for each class I am supporting right now, and more classes in the future.

Reply:
Maybe the code is easier to understand if all of the serialization code is together (less copy-paste elsewhere)? But maybe less in keeping of the GTSAM philosophy of a single test file per class. So I'm not sure, perhaps it's ok right now